builder/ibmcloud/vpc/config.go

```diff
@@ -82,7 +83,17 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 	}
 
 	if c.VSIBaseImageID == "" && c.VSIBaseImageName == "" {
-		errs = packer.MultiErrorAppend(errs, errors.New("a vsi_base_image_id or vsi_base_image_name must be specified"))
+		flag := false
+		if c.CatalogOfferingVersionCRN == "" {
+			errs = packer.MultiErrorAppend(errs, errors.New("a catalog_offering_version_crn must be specified if vsi_base_image_id or vsi_base_image_name are not present"))
```

Comment:
When we provide none `vsi_base_image_id` or `vsi_base_image_name` or `catalog_offering_version_crn`, message should be reflect clear need to use of them rather than proposing one of them.